decidim-admin/app/views/decidim/admin/taxonomies/_table.html.erb

```diff
@@ -12,6 +12,12 @@
       </th>
     </tr>
     </thead>
+    <%#
+     If you ever reach this template as source of inspiration for a table row drag and drop functionality,
+     please divert to a simpler solution, like the one in admin components table. This is a custom implementation as
+     it has the capability of nesting elements (a draggable within a draggable) so that you can reorder parents and children
+     Please note, it is not a functionality that allows you to move a child of a node, as child of another node.
```

Comment:
```suggestion
     If you ever reach this template as source of inspiration for a table row drag and drop functionality,
     please take in to account that this is intended to manage a complex case, as it has the capability of nesting elements (a draggable within a draggable) so that you can reorder parents and children.
     In Decidim there are two ways to implement a sortable (but both use the same library html5sortable.es).

     The simpler one is just to add the [data-draggable-table] attribute in the parent element, it requires to also add the [data-sort-url] attribute. This URL will be used to send a PUT request and ignore the result. 
      This implementation is used for instance in admin components table. 

     For cases when you need more control you can use the second implementation, which is the one used here.
     It uses the "js-sortable" class to initialize the element and allows you to use all the options documented in https://github.com/lukasoppermann/html5sortable as [data-sortable-OPTION] in the html.
     Take into account, that, if using the second option, you need to implement the "sortupdate" event by yourself (see  decidim-admin/app/views/decidim/admin/taxonomies/_table.html.erb for an example). This allows you to control the return of the request for instance.
```

I'd put a text more like this, but, to be honest, I think this belong to the documentation and put a link to it here  instead.